### ISSUE TITLE
Fix multi-column \contentoverview

### ISSUE THREAD

**Issue by Benno2000** [SoftVarE-Group/SlideTemplate#75]
When a table of contents is generated using the `\contentoverview` command, there are always 2 columns being used which results in a weird look for short TOCs. This should be modified in a way that multiple columns are only used if the TOC gets too long and that section titles are not displayed at the bottom of one column with the corresponding subsections at the top of the next column.

**Comment by tthuem** [comment]
I think it would be totally fine if the default is a single column and users can specify the number of columns if needed. Either with a parameter in the template or simply by overriding a command. I think the later would be sufficient here.

@EagleoutIce What do you think?

**Comment by EagleoutIce** [comment]
I agree, simply letting the user specify the number of columns should be sufficient (and easy to implement).

**Comment by tthuem** [comment]
@EagleoutIce As parameter of the template or by redefining a macro?

**Comment by EagleoutIce** [comment]
As this is guided by a simple number in the content overview Definition I would allow to pass the number of columns with a parameter.

**Comment by Benno2000** [comment]
Sounds good! I implemented this behaviour in https://github.com/SoftVarE-Group/SlideTemplate/commit/c374327d651b3c2e6871ed59ce31dd80e8d63c63. 

However, as this is a breaking change, I would not include this feature in the v0.3.0 release yet. Otherwise it could destroy existing slides that rely on the two-column contentoverviews. What do you think?

**Comment by EagleoutIce** [comment]
We could integrate the feature with `2` as a default into v0.3.0 and change that default with v1.0.0.

**Comment by tthuem** [comment]
I like the proposal by @EagleoutIce 

@Benno2000, please create a separate issue for the other milestone that we do not forget about this plan.

**Comment by Benno2000** [comment]
I also like the idea, but I'm not sure if this would cause confusion if one relies on 2 as the default and then suddenly it has to be changed everywhere with the next release.

**Comment by EagleoutIce** [comment]
But isn't that the definition of a breaking change?

**Comment by Benno2000** [comment]
That's right, then let's introduce the parameter with a temporary default of `2`. I'll create a pull request for that and also created an issue in v1.0.0 to change the default to `1` (https://github.com/SoftVarE-Group/SlideTemplate/issues/78).
